airbyte-db/lib/src/main/resources/configs_database/schema_dump.txt

```diff
@@ -1,3 +1,42 @@
+create table "public"."actor"(
+  "id" uuid not null,
```

Comment:
for these ids you can tell postgres to as a default behavior generate a uuid. this is what the sql looks like in postgres. not sure what needs to be done in jooq to do it.

```
"id" uuid PRIMARY KEY DEFAULT uuid_generate_v4(),
```

Reply:
Am trying to figure out how to do that in jooq. will update it

Reply:
any luck?